### PR TITLE
Run3 Suu Diquark cards (Madgraph5_aMC@NLO)

### PR DESCRIPTION
**Pull request by izisopou**
Dear experts,

I am uploading the Run3 cards for the Suu diquark signal used in the paired dijet analysis (EXO-21-010). 

This PR includes slightly modified cards with respect to the Run2 cards located in https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Suu_Diquark :

1) Center-of-mass energy adjusted to 6.8 TeV per beam

2) Typo nevents_for_max_weigth has been fixed in madspin card 

3) In run card replaced
 
   lhapdf    = pdlabel     ! PDF set                                     
   315200 = lhaid    ! if pdlabel=lhapdf, this is the lhapdf number
   
   with
   
   'lhapdf'    = pdlabel     ! PDF set                                     
   $DEFAULT_PDF_SETS    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number

4) In run card replaced

   50000 = nevents ! Number of unweighted events requested
   
   with
   
   1000 = nevents ! Number of unweighted events requested
   
   
Gridpacks are produced successfully with these new Run3 cards, along with a successful generation of an LHE file. However, there is a critical message in the log files [1] that we are not certain if it should be fixed and how.

Furthermore, in our log files we noticed that when the model import happens, it first fails [2], then converts the model (to python3) and tries the import again. We would therefore like to know if this is something expected or we should make any adjustments to the cards to prevent this from happening.

Thank you in advance.

Best regards,
Ilias (on behalf of the paired dijet group)

[1] CRITICAL: Model with non QCD emission of gluon (found 2 of those).
  This type of model is not fully supported within MG5aMC.
  Restriction on LO dynamical scale and MLM matching/merging can occur for some processes.
  Use such features with care.

[2] import model Suu_diquark_UFO
fail to load model but auto_convert_model is on True. Trying to convert the model
convert model /afs/cern.ch/work/i/izisopou/public/GenerateGridpacks/genproductions/bin/MadGraph5_aMCatNLO/Suu_Diquark_S8000_chi2000/Suu_Diquark_S8000_chi2000_gridpack/work/MG5_aMC_v2_9_9/models/Suu_diquark_UFO
retry the load of the model
import model Suu_diquark_UFO

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/Suu_Diquark_S8000_chi2000/Suu_Diquark_S8000_chi2000_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# set BW_cut 15                 # cut on how far the particle can be off-shell
+# set spinmode onshell          # Use one of the madspin special mode
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+# specify the decay for the final state particles
+decay t > w+ b, w+ > all all
+decay t~ > w- b~, w- > all all
+decay w+ > all all
+decay w- > all all
+decay z > all all
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/Suu_Diquark_S8000_chi2000/Suu_Diquark_S8000_chi2000_madspin_card.dat:31]: these lines are not needed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/template/SuuDQTemplate_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# set BW_cut 15                 # cut on how far the particle can be off-shell
+# set spinmode onshell          # Use one of the madspin special mode
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+# specify the decay for the final state particles
+decay t > w+ b, w+ > all all
+decay t~ > w- b~, w- > all all
+decay w+ > all all
+decay w- > all all
+decay z > all all
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/template/SuuDQTemplate_madspin_card.dat:31]: these lines are not needed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/Suu_Diquark_S8000_chi2000/Suu_Diquark_S8000_chi2000_madspin_card.dat (hunk 1 above)
Comment (izisopou) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/Suu_Diquark_S8000_chi2000/Suu_Diquark_S8000_chi2000_madspin_card.dat]: okay, I removed them

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/template/SuuDQTemplate_madspin_card.dat (hunk 2 above)
Comment (izisopou) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Suu_Diquark/template/SuuDQTemplate_madspin_card.dat]: okay, I removed them

**Comment by menglu21** [comment]
the cards look good to me now, @Saptaparna  @mlizzo @Dominic-Stafford please have a check.
btw, regarding the error message of the model, there are some discussion https://answers.launchpad.net/mg5amcnlo/+question/690133, it could be fine.

**Pull request merged**